ci-operator/step-registry/workers-rhel/vsphere-provision/workers-rhel-vsphere-provision-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: workers-rhel-vsphere-provision
+  from: upi-installer
+  commands: workers-rhel-vsphere-provision-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: "2"
+    documentation: |-
+      The number of rhel worker to be created.
+  - name: RHEL_IMAGE
+    documentation: |-
+      The name of rhel vm/template existing under folder /<Datacenter>/vm/Templates on vSphere, used to clone RHEL instances
+  - name: PLATFORM_VERSION
```

Comment:
Seem like the `RHEL_IMAGE` will have the rhel version info, so I guess we can drop it here?

Reply:
yeah, this ENV is not used in shell script after introducing RHEL_IMAGE, will remove it.